lib/linux/background-managers/cinnamon.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const childProcess = require('child_process');
+const util = require('util');
+const {existCommand} = require('../util');
+
+const execFile = util.promisify(childProcess.execFile);
+const exec = util.promisify(childProcess.exec);
+
+exports.available = async function () {
+	if (!await existCommand('gsettings')) {
+		return false;
+	}
+
+	try {
+		const {stdout} = await exec('gsettings list-schemas | grep org.cinnamon.desktop.background');
```

Comment:
Use `execFile` and do the grepping in JS instead of the shell. I would prefer to do the most amount of work on the JS side for maintainability reasons.